webapp/src/App.tsx

```diff
@@ -32,43 +32,50 @@ const SoundEffectTWO = () => {
 
 function App() {
   return (
-    <div className="App">
-      <NavBar />
+    <Auth0Provider
+      domain="ramsey-recipe-dev.us.auth0.com"
+      clientId="f0qoM6yhccCnMIHCjvlbSiQGQNWXc3Ii"
```

Comment:
_normally_ I would say to put this in an env file for easier configuration in the future, even though it's not strictly secret it's still a configuration.

But you've done that on fire damper and this is a temp project.